controller/error.go

```diff
@@ -25,15 +25,6 @@ func IsInvalidConfig(err error) bool {
 	return microerror.Cause(err) == invalidConfigError
 }
 
-var invalidRESTClientError = &microerror.Error{
```

Comment:
This became useless. The errors I saw for such cases became better meanwhile. Should be fine in case something goes wrong in this direction.